tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -173,6 +203,17 @@ def _consume_save_queue(self):
         else:
             self.save_errors = 0
 
+    def get_type_from_key(self, key: str) -> str:
+        return key.split()[0]
+
+    def _serialize_item(self, key: str, state: Dict[str, Any]) -> str:
```

Comment:
if this only accepts two values for `key`, i think we can write `key: Literal[runstate.JOB_STATE, runstate.JOB_RUN_STATE]` rather than `key: str`

Reply:
I did have that, but was getting `Variable not allowed in type expression`. I guess in this case I should have done `Literal[job_state", "job_run_state"]` or maybe just pretended it's an enum and added `# type: ignore`?